src/services/ApiCalls.tsx

```diff
@@ -0,0 +1,63 @@
+import { URL_SERVER } from "constants/serverConnexion";
+
+const calculateAge = (birthday: any) => {
+  var ageDifMs = Date.now() - birthday;
+  var ageDate = new Date(ageDifMs); // miliseconds from epoch
+  return Math.abs(ageDate.getUTCFullYear() - 1970);
+};
+interface Patient {
+  firstName: String;
+  lastName: String;
+  age: number;
+  id: String;
+}
+
+export const CallApi = (typeR: string, queryR: any) => {
+  let mkFhir = require("fhir.js");
+  let client = mkFhir({
+    baseUrl: URL_SERVER
+  });
+
+  let patientPromise = client.search({ type: typeR, query: queryR }); // limited to 20 patients ?
+  return patientPromise;
+};
+
+export const getPatients = () => {
+  let promise = CallApi("Patient", { _count: 100, _page: 1 })
+    // The server should have a page option, will allow to show pages on the PatientTable
+    // see https://www.hl7.org/fhir/search.html#count
+    .then(function(res: any) {
```

Comment:
Could you use the async/await syntax instead of `.then()`?

Reply:
Done